src/arrayMethodReduce.js

```diff
@@ -4,8 +4,20 @@
  * Implement method Reduce
  */
 function applyCustomReduce() {
-  [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+  [].__proto__.reduce2 = function(accumulator, ...startValue) {
+    let result = this[0];
+    let start = 1;
+
+    if (startValue.length > 0) {
+      result = startValue[0];
+      start = 0;
+    }
```

Comment:
```suggestion
    if (startValue.length >= 2) {
      result = initialValue;
      start = 0;
    }
```

Reply:
Hello! Could you explain me this comment?
If we change '...startValue' to 'initialValue' at line 7, where will we get the 'startValue' for line 11?